src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/ObjectWriter/R2RPEBuilder.cs

```diff
@@ -710,11 +710,14 @@ public static PEHeaderBuilder Create(Characteristics imageCharacteristics, DllCh
                 sectionAlignment = fileAlignment;
             }
 
-            dllCharacteristics &= (DllCharacteristics.NxCompatible | DllCharacteristics.TerminalServerAware | DllCharacteristics.AppContainer);
+            dllCharacteristics &= ~DllCharacteristics.AppContainer;
```

Comment:
Do we have legitimate cases when the `AppContainer` flag is set?

Reply:
No idea, more or less mechanically copied over from

https://github.com/dotnet/runtime/blob/97e553f5c3244b7df65ebfb7de3bb712081228cc/src/coreclr/src/zap/zapimage.cpp#L1520

Reply:
Hmm, sorry about this, I think I have messed this up, let me fix this...

Reply:
I misread the code, the bit negation operator (<code>~</code>) shouldn't be there. I have updated the PR, please take another look when you have a chance; thanks for pointing this out!

Reply:
Looks good, thank you.